### ISSUE TITLE
DynamicMarkingStratumViewModel

### ISSUE THREAD

**Issue by jsbean** [jsbean/DNM#188]
Once `DynamicMarkingStratumFactory` manages `ComponentSpans`...

**Comment by jsbean** [comment]
Actually, before may be cleaner.

**Comment by jsbean** [comment]
```Swift
struct DynamicMarkingSpannerModel {
    let direction: Int // -1, 0, 1
    let exponent: Float
    // or
    // let controlPoints: [(Float, Float)] // ?
    // later: 
    let truncateBegin: Float? 
    let truncateEnd: Float?
    // later:
    let sections: [...] 
}
```

**Comment by jsbean** [comment]
Closed with dfc6923a7a4729b3af6a0c00cb8f31876cd7aa11
